### PR TITLE
Copy OTP button added to the search results

### PR DESCRIPTION
**Pull request by myelsukov**
Changes to be committed:
	modified:   chrome/background.browserify.js
	new file:   chrome/icon-otp.svg
	modified:   chrome/script.browserify.js
	modified:   chrome/styles.css
Icon is pulled from https://github.com/danklammer/bytesize-icons/blob/master/dist/icons/clock.svg

### CODE REVIEW

**Comment by erayd** [comment]
@maximbaz What if we added a right-click menu, which loaded the file *before* displaying, so that we know which options are available?

We can't decrypt every file to figure out which buttons to display (because decryption is slow), but we can certainly decrypt one.

**Comment by maximbaz** [comment]
I like that, noted for v3 😉

File: chrome/script.browserify.js [hunk 1]
```diff
@@ -93,6 +93,11 @@ function view() {
             onclick: loginToClipboard.bind({ entry: login, what: "password" }),
             title: "Copy password",
             tabindex: -1
+          }),
+          m("button.copy.otp", {
+              onclick: loginToClipboard.bind({ entry: login, what: "otp" }),
+              title: "Copy OTP",
```
[the remark this answers is not in the document]
Comment (myelsukov) [reply chrome/script.browserify.js]: I absolutely don't mind. I am back-end engineer, so historically I don't care about UI :)

**Comment by myelsukov** [comment]
Suggested changes have been implemented. Please review.

**Comment by myelsukov** [comment]
My pleasure!

**Pull request merged**